src/mame/drivers/twincobr.cpp

```diff
@@ -1236,46 +1236,97 @@ ROM_START( fnshark ) // based on a different version of the game code? (only a ~
-	ROM_LOAD( "82s129.ic41",  0x000, 0x100, CRC(24e7d62f) SHA1(1c06a1ef1b6a722794ca1d5ee2c476ecaa5178a3) ) /* bprom type: 82s129AN - sprite priority control ?? */
-	ROM_LOAD( "82s129.ic40",  0x100, 0x100, CRC(a50cef09) SHA1(55cafb5b2551b80ae708e9b966cf37c70a16d310) ) /* bprom type: 82s129AN - sprite priority control ?? */
-	ROM_LOAD( "82s123.ic42",  0x200, 0x020, CRC(f72482db) SHA1(b0cb911f9c81f6088a5aa8760916ddae1f8534d7) ) /* bprom type: 82s123AN - sprite control ?? */
-	ROM_LOAD( "82s123.ic50", 0x220, 0x020, CRC(bc88cced) SHA1(5055362710c0f58823c05fb4c0e0eec638b91e3d) ) /* bprom type: 82s123AN - sprite attribute (flip/position) ?? */
-	ROM_LOAD( "82s123.ic99", 0x240, 0x020, CRC(4fb5df2a) SHA1(506ef2c8e4cf45c256d6831a0a5760732f2de422) ) /* bprom type: 82s123AN - tile to sprite priority ?? */
+	ROM_REGION( 0x300, "proms", 0 ) // Nibble BPROMs, lo/hi order to be determined
+	ROM_LOAD( "82s129.ic41", 0x000, 0x100, CRC(24e7d62f) SHA1(1c06a1ef1b6a722794ca1d5ee2c476ecaa5178a3) ) // BPROM type: 82s129AN - sprite priority control ??
+	ROM_LOAD( "82s129.ic40", 0x100, 0x100, CRC(a50cef09) SHA1(55cafb5b2551b80ae708e9b966cf37c70a16d310) ) // BPROM type: 82s129AN - sprite priority control ??
+	ROM_LOAD( "82s123.ic42", 0x200, 0x020, CRC(f72482db) SHA1(b0cb911f9c81f6088a5aa8760916ddae1f8534d7) ) // BPROM type: 82s123AN - sprite control ??
+	ROM_LOAD( "82s123.ic50", 0x220, 0x020, CRC(bc88cced) SHA1(5055362710c0f58823c05fb4c0e0eec638b91e3d) ) // BPROM type: 82s123AN - sprite attribute (flip/position) ??
+	ROM_LOAD( "82s123.ic99", 0x240, 0x020, CRC(4fb5df2a) SHA1(506ef2c8e4cf45c256d6831a0a5760732f2de422) ) // BPROM type: 82s123AN - tile to sprite priority ??
+ROM_END
+
+ROM_START( skysharbl )
+	ROM_REGION( 0x30000, "maincpu", 0 ) // Main 68K code
+	ROM_LOAD16_BYTE( "1r.ic18", 0x00000, 0x10000, CRC(ea4bcb43) SHA1(4b5fda235908a9081fdd4cca98294e9e9a34bbf2) )
+	ROM_LOAD16_BYTE( "1q.1ic7", 0x00001, 0x10000, CRC(d1f39ed2) SHA1(2a46a542c29a07b31a6bfa72a2f4d4d26699c13d) )
+
+	ROM_REGION( 0x8000, "audiocpu", 0 ) // Sound Z80 code
+	ROM_LOAD( "1p.ic16", 0x0000, 0x8000, CRC(f0b98af2) SHA1(7054029b1955c510a6b693d278dd4d8a384112df) )
+
+	ROM_REGION( 0x0c000, "gfx1", 0 )    // Chars
+	ROM_LOAD( "1g.ic7", 0x00000, 0x04000, CRC(9d3f698d) SHA1(8e5497929663ec3bd27e9a84fe068d12c53de5c5) ) // 1xxxxxxxxxxxxx = 0xFF
+	ROM_LOAD( "1e.ic5", 0x04000, 0x04000, CRC(543bbb81) SHA1(8aabc3d4f14b6531af000008b50cf09bb6cd003f) ) // 1xxxxxxxxxxxxx = 0xFF
+	ROM_LOAD( "1f.ic6", 0x08000, 0x04000, CRC(d357f494) SHA1(e83560e551ea15cac903e9a8c5ce57d51af175c7) ) // 1xxxxxxxxxxxxx = 0xFF
+
+	// Same data on larger EPROMs with first half empty
+//  ROM_LOAD( "5.bin", 0x08000, 0x04000, NO_DUMP ) // Not dumped on this set
+//  ROM_CONTINUE(0x08000,0x04000)
+//  ROM_LOAD( "6.bin", 0x04000, 0x04000, NO_DUMP ) // Not dumped on this set
+//  ROM_CONTINUE(0x04000,0x04000)
+//  ROM_LOAD( "7.bin", 0x00000, 0x04000, NO_DUMP ) // Not dumped on this set
+//  ROM_CONTINUE(0x00000,0x04000)
```

Comment:
So what’s going on here?  Do these ROMs actually exist undumped?  If they are, I’m not sure it’s helpful to add the game in the current state.  There seem to be an increasing number of games added where we just hope certain ROMs are the same as other sets.  On the other hand, if these ROMs don’t actually exist, the comment is unhelpful.

Reply:
Upps! No. I was thinking on them when Jordi told me that he left four (four, not three) ROMs undumped, but those are for the MCU code, not GFX.
I've removed the comment and add a note about the undumped MCU code.